sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -381,7 +381,7 @@ def test_lambertw():
     assert LambertW(2, evaluate=False).is_real
     p = Symbol('p', positive=True)
     assert LambertW(p, evaluate=False).is_real
-    assert LambertW(p - 1, evaluate=False).is_real is None
+    assert LambertW(p - 1, evaluate=False).is_real is False
```

Comment:
That's how as_coeff_Add works. The first term is always an integer or rational number.